WalletWasabi.Fluent/Models/UI/UiClipboard.cs

```diff
@@ -11,8 +11,9 @@ public async Task<string> GetTextAsync()
 	{
 		if (ApplicationHelper.Clipboard is { } clipboard)
 		{
-			return await clipboard.GetTextAsync();
+			return await clipboard.GetTextAsync() ?? "";
```

Comment:
I think we don't use `ConfigureAwait(false)` in UiModels and ViewModels.

Reply:
The usage of `ConfigureAwait(false)` doesn't depend on ViewModels or Models, it depends on what code is executed after the awaited call.

For example:

```c#
public async Task MethodA()
	{
		await MethodB();
		// Here are some UI related code executed afterward.
	}

public async Task MethodB()
	{
		await MethodC().ConfigureAwait(false);
		// NO UI related code executed afterward.
	}
```

After awaiting `MethodB()` you cannot add `ConfigureAwait(false)` because after it you need to stay on the UI thread as you deal with objects that must be changed on the UI thread.
While in `MethodB()` you can safely add it as after calling `MethodC()` you don't have to stay on UI thread as there is no UI related code executed.

Reply:
> Shouldn't be there `ConfigureAwait(false)`? Similarly elsewhere.

Yes, it should be there. And TBH I don't think that it was added anywhere else in the UI, so improving the performance, it would be worth checking and fixing it if needed.